.gitignore

```diff
@@ -8,6 +8,7 @@ assets/hashes.js
 backend_prod_files/*
 core/templates/prod/*
 core/tests/.browserstack.env
+node_modules/*
 .coverage*
```

Comment:
Should we be adding `package-lock.json` to gitignore? There are mixed responses to these. What do you suggest?

Reply:
This cannot be added into `.gitignore` ([ref](https://docs.npmjs.com/files/package-locks#using-locked-packages)) but I was thinking can we make this as a binary file (like an image)?

Reply:
Yeah, I did look into this, and my sense is that we should leave it checked-in and in its current form.

Reply:
Actually, there can be cases where this file creates merge conflicts ([ref](https://docs.npmjs.com/files/package-locks#resolving-lockfile-conflicts)). Therefore, ignoring was a possible solution but I am fine with committing this as well. Thanks!